more/free-programming-interactive-tutorials-en.md

```diff
@@ -117,6 +120,11 @@
 * [Smooth CoffeeScript, Interactive Edition](http://autotelicum.github.io/Smooth-CoffeeScript/interactive/interactive-coffeescript.html)
 
 
+### cmd
+
+* [Command Challenge](https://cmdchallenge.com) - Command Challenge
```

Comment:
cmd is not a thing.  this is linux/unix.
the resource fits better in the playgrounds list